forwards_paper.tex

```diff
@@ -394,40 +399,40 @@ \subsection*{Recording the pedigree in forwards time}
 we need to record two things for each new chromosome:
 the birth time, and the endpoints and parental IDs of each distinctly inherited segment,
 which are naturally stored as the \emph{nodes} and \emph{edges} of a tree sequence.
-For concreteness, here we write out in pseudocode how to run a neutral Wright--Fisher simulation
-(but with overlapping generations) that records genealogical history in this way.
+For concreteness, we write out in pseudocode how to run a neutral Wright--Fisher simulation
+that records genealogical history in this way.
 The simulation will run for $T$ generations,
-and has $N$ haploid individuals, each carrying a single chromosome of length L.
-For simplicity we sample exactly one crossover per generation.
-The probability of death per individual each generation is $\delta$.
-We will use
-$\randomuniform(A)$ to denote an element of the set $A$ chosen uniformly at random
-(and each such instance represents and independent draw). Given a node table
-$\Nt$, the function $\taddnode{\Nt, t}$ adds a new node to the table with time $t$
-and returns the ID of this new node. Similarly, the function $\taddedge{\Et,
-\ell, r, p, c}$ adds a new edge $(\ell, r, p, c)$ to the edge table $\Et$.
-The function $\tsimplify{P, \Nt, \Et}$ simplifies the history stored in the
-tables $\Nt$ and $\Et$ to the minimal information required to represent the
-genealogies of the list of node IDs $P$ (see section XXX).
-We first list
-the algorithm in detail, and immediately follow this with a step-by-step
-explanation.
+and has $N$ haploid individuals, each carrying a single chromosome of length $L$.
+For simplicity, we sample exactly one crossover per generation.
+
+We use $\randomuniform(A)$ to denote an element of the set $A$ chosen uniformly at random.
+% (and all such instances is independent).  % this is understood here
```

Comment:
I thought this was obvious too, but  Alison and Amandine were both confused by this notation. I think it's probably safer to leave in the independence comment .

Reply:
I suspect not "confused" but "pedantic"... I usually am, also. =)  Sounds good.